packages/frontend/tests/versus_match_card.test.tsx

```diff
@@ -0,0 +1,83 @@
+import { render } from "@testing-library/react";
+import { VersusMatchCard } from "../src/components/versus_match_card";
+
+/* Jest + React Guides: https://jestjs.io/docs/tutorial-react */
+
+// Group of related tests.
+describe("VersusMatchCard", () => {
+  //     \/ Describe individual test here.
+  test("Correct scores displayed", () => {
+    const score = {
+      Lara: 10,
+      Hugo: 1,
+    };
+
+    const component = render(<VersusMatchCard score={score} />);
+    expect(component.container).toBeInTheDocument();
+
+    const matchCard = component.container.children.item(0) as HTMLElement;
+    expect(matchCard).not.toBeNull();
+
+    const div1 = matchCard.children.item(0) as HTMLElement;
+    const div2 = matchCard.children.item(2) as HTMLElement;
+
+    expect(div1).not.toBeNull();
+    expect(div2).not.toBeNull();
+
+    const p11 = div1.children.item(0);
+    expect(p11).not.toBeNull();
+    expect(p11!.textContent).toBe("Lara");
```

Comment:
Linting error. As you are not using vscode, please make sure to run `yarn lint:fix` before pushing and fixing any issues.

In this case it can just be swapped to a `?` as we don't care about the type.

Also as a subnote, we prefer tests failing instead of crashing! The null assertion would crash the tests (meaning no other tests can run) which is really hard to debug

```suggestion
    expect(p11?.textContent).toBe("Lara");
```